jhub_apps/templates/style.css

```diff
@@ -274,6 +276,9 @@ a:focus {
     background-color: {{ navbar_hover_color | default("#00a3b0") }};
 }
 
+.MuiButtonBase-root.MuiListItemButton-root {
+}
+
```

Comment:
This doesn't appear to be doing anything, remove if not.

Reply:
This was supposed to have the --primary-color-light: {{ primary_color_light | default("#ba18da10") }}; as its styling.

I